### ISSUE TITLE
Dropout for Convolution layers

### ISSUE THREAD

**Issue by xingyul** [ppwwyyxx/tensorpack#244]
The Dropout operation on convolution layers should drop the whole feature map, i.e. set noise_shape parameter of [tf.nn.dropout](https://www.tensorflow.org/api_docs/python/tf/nn/dropout).

**Comment by ppwwyyxx** [comment]
You can just use tf.nn.dropout in your model code with the argument you like.
